src/N7consulting/PrivacyBundle/Controller/PrivacyController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace N7consulting\PrivacyBundle\Controller;
+
+use Mgate\PersonneBundle\Entity\Personne;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class PrivacyController extends Controller
+{
+
+    /** GDPR actions */
+    public const ACCESS = 'access';
```

Comment:
does it need to be public? Should be `private` IMO. Also you could rename them `GDPR_ACCESS_ACTION` for example instead of requiring that comment

Reply:
Name is indeed much better, thanks.

Those constants are public I now use them in the homepage template.